cmd/tptctl/cmd/config_aws_cloud_account.go

```diff
@@ -126,11 +132,6 @@ var ConfigAwsCloudAccountCmd = &cobra.Command{
 			if err != nil {
 				cli.Error("failed to delete aws account", err)
 			}
-
-			err = provider.DeleteResourceManagerRole(roleName, *awsConf)
-			if err != nil {
-				cli.Error("failed to delete threeport role", err)
-			}
```

Comment:
This delete operation didn't need to be here, as the role wouldn't exist in the event that this code is executed.